app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -56,8 +56,8 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               required
               className="w-full flex-grow-0"
               label="Assigned Contact"
-              name="assigned contact"
-              options={ROLE}
+              name="role"
+              options={ROLES}
               placeholder="Select Role"
```

Comment:
I still don't understand. The label uses the keyword `Contact`. But the placeholder and variables use the keyword `Role`. Can you please point out how both of these are associated?

Reply:
I referred the figma design, it had Assigned Contact as sub heading & placeholder as "Select the Role".
Should I modify it ?

Reply:
Thanks for pointing that out. I will take care of updating the figma design.